sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -706,11 +699,36 @@ export class JsonRpcProvider extends Provider {
     return this.wsClient.unsubscribeEvent(id);
   }
 
-  async devInspectTransaction(txBytes: string): Promise<DevInspectResults> {
+  async devInspectTransaction(
+    sender: SuiAddress,
+    tx: UnserializedSignableTransaction | string | Base64DataBuffer,
+    gasPrice: number | null = null,
+    epoch: number | null = null
+  ): Promise<DevInspectResults> {
     try {
+      const version = await this.getRpcApiVersion();
+      // TODO: remove after 0.23.0 is deployed in both DevNet and TestNet
+      if (version?.major == 0 && version?.minor < 23) {
+        return this.devInspectTransactionDeprecated(sender, tx, epoch);
```

Comment:
@tnowacki We are going to release a new DevNet on Jan 25th(Wednesday). If this is not merged before the DevNet release, then we would need to update `version?.minor < 23` to `version?.minor < 24`.

Reply:
When is the cut for 0.23?